docs/ARCHITECTURE.md

```diff
@@ -87,7 +87,7 @@ for another platform.
 
 These files are mostly only relevant when running crates.io's code in development mode.
 
-* `bors.toml` - Configure our instance of [bors-ng][] for continous integration
+* `bors.toml` - Configure our instance of [bors-ng] for continous integration
```

Comment:
Oops-- we haven't been using bors-ng for a while, but we forgot to remove the config files. [I've opened a PR to fix that](https://github.com/rust-lang/crates.io/pull/1944) which will probably cause a merge conflict here; you can discard these changes.

Thank you so much for remembering to update the documentation now so that we don't have this problem again in the future!